### PR TITLE
RVIZ Panel for Speedometer and Wheel

### PR DESCRIPTION
**Pull request by mukundSai**
I made a panel for visualizing the speed and angle of the wheel.

### CODE REVIEW

File: rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+#include <rr_platform/chassis_state.h>
+#include <rr_platform/steering.h>
+#include <QLabel>
+#include <QtGui>
+#include <QPainter>
+#include <math.h>
+#include <QString>
+
+
+/*
```
Comment (JasonGibson274) [inline rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h]: maintainer preference but I would remove the comments that came with the example panel now that you implemented something.

Comment (barulicm) [reply rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h]: Yes, @mukundSai ^

**Comment by JasonGibson274** [comment]
It also looks like you have not configured git correctly on your machine

File: rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+
+namespace rr_rviz_plugins {
+
+
+class SpeedometerPanel : public rviz::Panel {
+
+
+
```
Comment (chsahit) [inline rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h]: Just a small note on style, we don't need this much whitespace between class declaration and then starting to populate the class. Ditto throughout the rest of these changes where there are just chunks of whitespace where comments used to be.

Comment (ebretl) [reply rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h]: @chsahit do you still think this needs to be fixed before I approve this PR?

**Pull request closed without merging**